src/pages/ValidateLoginPage/index.js

```diff
@@ -0,0 +1,59 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import {
+    propTypes as validateLinkPropTypes,
+    defaultProps as validateLinkDefaultProps,
+} from './validateLinkPropTypes';
+import * as User from '../../libs/actions/User';
+import FullScreenLoadingIndicator from '../../components/FullscreenLoadingIndicator';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as Session from '../../libs/actions/Session';
+import Permissions from '../../libs/Permissions';
+import Navigation from '../../libs/Navigation/Navigation';
+
+const propTypes = {
+    /** The accountID and validateCode are passed via the URL */
+    route: validateLinkPropTypes,
+
+    /** List of betas available to current user */
+    betas: PropTypes.arrayOf(PropTypes.string),
+};
+
+const defaultProps = {
+    route: validateLinkDefaultProps,
+    betas: [],
+};
+
+class ValidateLoginPage extends Component {
+    componentDidMount() {
+        if (Permissions.canUsePasswordlessLogins(this.props.betas)) {
+            if (this.isAuthenticated()) {
+                Navigation.goBack(false);
```

Comment:
Looking at the index.website.js file, I noticed the difference is that we're calling `Navigation.goBack(false);`, I think we should add a comment explaining why we're doing this and why this file is different.

Reply:
👍 
fixed.